test/user_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require 'pry'
+
+class UserTest < Minitest::Test
+
+	def setup
+		@sal = User.new("Sal")
+		@ali = User.new("Ali")
+	end
+
+	def test_user_exists
+		assert_instance_of User, @sal
+	end
+
+	def test_user_attributes
+		assert_equal @sal.name, "Sal"
+		assert_equal @sal.jokes, []
+	end
+
+	def test_learn
+		joke_1 = Joke.new(1, "Why did the strawberry cross the road?",\
+ "Because his mother was in a jam.")
+		joke_2 = Joke.new(2, "How do you keep a lion from charging?",\
+ "Take away its credit cards.")
+		@sal.learn(joke_1)
+		@sal.learn(joke_2)
+		assert_equal @sal.jokes.count, 2
```

Comment:
same comment here-- assert an array of the actual jokes you want, not just the count.